scripts/install_environment.sh

```diff
@@ -21,9 +21,43 @@ function install_libvirt() {
         libvirt-devel \
         libvirt-daemon-kvm \
         qemu-kvm \
-        libgcrypt \
-        swtpm \
-        swtpm-tools
+        libgcrypt
+
+    # TODO: get swtpm RPMs from EPEL as soon as 0.7.1 or later lands there.
+    # based on: https://github.com/stefanberger/swtpm/wiki#compile-and-install-on-linux
+
+    echo "Enable EPEL"
+    dnf install -y  epel-release
+
+    echo "Install swtpm dependencies"
+    sudo dnf install -y \
+        libtasn1-devel \
+        expect \
+        socat \
+        python3-twisted \
+        fuse-devel \
+        glib2-devel \
+        gnutls-devel \
+        gnutls-utils \
+        gnutls \
+        json-glib-devel \
+        autoconf \
+        automake \
+        libtool \
+        openssl-devel \
+        libtpms \
+        libtpms-devel \
+        libseccomp-devel \
+        net-tools \
+        iproute \
+        git \
+        make
+
+    echo "Install swtpm v0.7.1 from source"
+    git clone --depth 1 --branch v0.7.1 https://github.com/stefanberger/swtpm.git
+    pushd swtpm
+    ./autogen.sh --with-openssl --prefix=/usr
+    popd
```

Comment:
how much time did it add this whole installation process?
because we might consider doing it specifically for tpm flow

Reply:
checking

Reply:
based on the job history, [job duration for successful runs](https://prow.ci.openshift.org/job-history/gs/origin-ci-test/pr-logs/directory/pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-tpmv2) was not increased.

Reply:
the added 'time dnf install' commands (enable EPEL + install RPM deps) did not show the cmd duration; however, the swtpm install commands did:

swtpm install commands:
real	0m1.163s
real	0m8.621s
real	0m5.370s
real	0m0.607s

15.758 seconds in total.